src/SkeletalTrapezoidation.cpp

```diff
@@ -847,7 +843,7 @@ void SkeletalTrapezoidation:: filterUnmarkedRegions(const BeadingStrategy& beadi
             continue;
         }
         assert(edge.to->data.bead_count >= 0 || edge.to->data.distance_to_boundary == 0);
-        coord_t max_dist = 400; // beading_strategy.getTransitioningLength(edge.to->data.bead_count)
+        coord_t max_dist = 400;
```

Comment:
Magic numbers everywhere! I think they should be `constexpr`d, or at the _very_ least `const`d.

I also see that this is a parameter passed by copy. Which is OK for something like a `coord_t`, but once you start passing things by copy it's easy to make a mistake. Maybe we should do a sweep of everything that is passed by copy to see if no objects or the like are unnecessarily copied.

Reply:
We have a `NoCopy` class for that. If you don't want a particular data structure to be copied, make it inherit from `NoCopy` and the compiler will tell you where it's being copied.

Reply:
That's good to know!

(Still leaves the original issue that magic constants should be `constexpr` as much as possible).